json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -185,6 +185,15 @@ public function get_settings_response() {
 				);
 
 				//allow future versions of this endpoint to support additional settings keys
+				/**
+				 * Filter the current site setting in the returned response.
+				 *
+				 * @module json-api
+				 *
+				 * @since 4.0.0
```

Comment:
This can't be since 4.0.0, please use the version number when that code actually was (will be) released. Thanks!